tests/components/recorder/test_init.py

```diff
@@ -1002,54 +1002,48 @@ def test_statistics_runs_initiated(hass_recorder):
             ) - timedelta(minutes=5)
 
 
-def test_compile_missing_statistics(tmpdir):
+@pytest.mark.freeze_time("2022-10-13 09:00:00+02:00")
```

Comment:
I suggest that you use a time not so close to the time of the PR.
We have seen issues where it was OK for the day of the PR, and then started failing the next day...

```suggestion
@pytest.mark.freeze_time("2022-09-13 09:00:00+02:00")
```